apps/docs/content/components/DuotoneText/index.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: DuotoneText
+description: Use the duotone text component to style text with an emphasized portion.
+---
+
+import ComponentLayout from '../../../src/layouts/component-layout'
+
+import anatomy from './images/anatomy.png'
+import doEmphasis from './images/do-emphasis.png'
+import dontEmphasis from './images/dont-emphasis.png'
+
+export default ComponentLayout
+
+<img
+  src={anatomy}
+  alt="A heading using the duotone text treatment to highlight some specific text. The heading reads 'GitHub Codespaces offers a complete dev environment in seconds.' with 'GitHub Codespaces' emphasized in a different color"
+/>
+
+## Usage
+
+Use duotone text to create a more dynamic reading flow by emphasizing words at the beggining of a text. Duotone text is a purely visual treatment and should not be used to convey semantic emphasis.
+
+<DoDontContainer>
+  <Do>
+    <img alt="" src={doEmphasis} />
+    <Caption>Only emphasize the first few words of a text.</Caption>
+  </Do>
+  <Dont>
+    <img alt="" src={dontEmphasis} />
+    <Caption>Don't emphasize other parts of the text.</Caption>
+  </Dont>
+</DoDontContainer>
+
+### Supported components
+
+We recommend using duotone text exclusively on components that have been designed to support this treatment:
```

Comment:
It's going to be really difficult to enforce this IMO, as you can use DuotoneText inside any Text or Heading component. Is this something we need to be careful about? The alternative is to only enable this pattern to work inside of the components you've mentioned.